examples/acmetool/responses

```diff
@@ -0,0 +1,15 @@
+# https://hlandau.github.io/acme/userguide#response-files
+#
+# This is a example of a response file, used with --response-file.
+# It automatically answers prompts for unattended operation.
+# grep for UniqueID in the source code for prompt names.
+# Pass --response-file to all invocations, not just quickstart.
+# If you don't pass --response-file, it will be looked for at "(state-dir)/conf/responses".
+# You will typically want to use --response-file with --stdio or --batch.
+# For dialogs not requiring a response, but merely acknowledgement, specify true.
+# This file is YAML. Note that JSON is a subset of YAML.
+"acme-enter-email": "hostmaster@example.com"
```

Comment:
Haven't found a solution for this yet. https://github.com/hlandau/acme/issues/243

From [acme/register.go:74](https://github.com/hlandau/acme/blob/83d2466ceb01a76d281ff8983465db191fbfa1e2/solver/register.go#L74): "... Although entering an e. mail address is optional, it is highly recommended."